stdlib/public/runtime/Errors.cpp

```diff
@@ -299,6 +303,34 @@ reportOnCrash(uint32_t flags, const char *message)
 #else
   // empty
 #endif // SWIFT_HAVE_CRASHREPORTERCLIENT
+
+#if defined(_WIN32)
+  // The last fatal error message is accessible for crash reporting on
+  // Windows.
+  char *oldMessage = nullptr;
```

Comment:
We could reuse the same code as above by defining `char** lastFatalErrorMessage = ` `&gCRAnnotations` or `&gLastFatalErrorMessage` based on preprocessor values.

Reply:
Done